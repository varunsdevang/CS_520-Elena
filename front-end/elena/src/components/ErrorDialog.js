import DialogTitle from '@mui/material/DialogTitle';
import Dialog from '@mui/material/Dialog';
import DialogActions from '@mui/material/DialogActions';
import DialogContent from '@mui/material/DialogContent';
import DialogContentText from '@mui/material/DialogContentText';
import Button from '@mui/material/Button';
import { Typography } from '@mui/material';

const ErrorDialog = (props)=>{
    return (
    <Dialog open={props.open} onClose={props.onClose} maxWidth="md">
      <DialogTitle>
        Error!
      </DialogTitle>
      <DialogContent>
        <DialogContentText>
        <Typography color="red">
            {props.message}
        </Typography>
        </DialogContentText>
      </DialogContent>
      <DialogActions>
        <Button onClick={props.onClose} color="primary">
          close
        </Button>
      </DialogActions>
    </Dialog>
    );
}

export default ErrorDialog;