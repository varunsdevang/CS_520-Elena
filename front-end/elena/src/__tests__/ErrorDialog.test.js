import { render, screen } from '@testing-library/react';
import React from "react";
import renderer from "react-test-renderer";
import ErrorDialog from '../Components/ErrorDialog';


describe("ErrorDialog snapshot should match", () => {
    it("Matches DOM Snapshot", () => {
      const domTree = renderer.create(< ErrorDialog/>).toJSON();
      expect(domTree).toMatchSnapshot();
    });
});