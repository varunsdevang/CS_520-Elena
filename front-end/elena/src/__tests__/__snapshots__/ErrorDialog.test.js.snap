// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorDialog snapshot should match Matches DOM Snapshot 1`] = `null`;
